src/routers/product-router.js

```diff
@@ -1,4 +1,8 @@
 const express = require('express');
 const router = express.Router();
 
+const { ProductModel } = require('../models');
+const { pageinatedResults} = require('../controllers/product-controller');
+
+router.get('/products', pageinatedResults(ProductModel))
```

Comment:
this router is already attached to /products prefix, so here you need to write only '/'